### PR TITLE
[full-ci] chore: bump web to v8.0.0-alpha.1

### PR DESCRIPTION
**Pull request by fschade**
## Description
bring web master and ocis master together

### CODE REVIEW

File: .drone.env
Comment (JanAckermann) [inline .drone.env:3]: Shouldn't that be master ?

Comment (fschade) [reply .drone.env:3]: it should, but i wanna see that it is green here before we merge web to master... hust in case, once it is green, ill change it back to master

**Comment by fschade** [comment]
please do not merge, it contains a temporary drone build feature which pulls web and build it.
we have a chicken egg problem here again, to get green it needs a non released web version, we could release a web version and use it but we will run into similar situations in the future too.

@micbar can we discuss what we could do? My idea, be able to build web based on a commit id and branch, once we have a web release we can fall back to the asset download again!

@phil-davis can you have a look, i have not enough knowledge about `owncloud/owncloud-test-middleware`

```
Error: Error: trashbin data not defined
          at /usr/src/app/src/helpers/webdavHelper.js:169:22
          at Array.forEach (<anonymous>)
          at /usr/src/app/src/helpers/webdavHelper.js:167:21
          at processTicksAndRejections (node:internal/process/task_queues:96:5)
          at /drone/src/webTestRunner/tests/acceptance/stepDefinitions/middlewareContext.js:38:15
          at processTicksAndRejections (node:internal/process/task_queues:95:5)
```

in `webUIFilesSearch/search.feature:60`, THX

**Comment by micbar** [comment]
We could use a daily web build as we also do for oc10 core in the apps repos.

**Comment by individual-it** [comment]
> We could use a daily web build as we also do for oc10 core in the apps repos.

I don't think we want to do that, what we want is to test the web that is delivered in the ocis binary and not an external one, because that is what the user will most likely use when starting ocis.

test what you fly - fly what you test :rocket:

File: .drone.env
Comment (individual-it) [reply .drone.env:3]: This commit ID is only to get the tests from web, we don't pull the web assets from that branch/commit. We want to test the web version that is currently build-in into ocis.
see also https://github.com/owncloud/ocis/pull/7198#issuecomment-1704607164

**Comment by fschade** [comment]
thanks michael, thanks artur!

i get your point `test what you fly - fly what you test` maybe we just have to release web more often, the point is, if you work on both it adds a lot of extra time.

**Comment by fschade** [comment]
web release as v8.0.0-alpha.1, now the ci has to agree...

**Comment by fschade** [comment]
web uses `owncloud/owncloud-test-middleware:1.8.3` (green) and ocis `owncloud/owncloud-test-middleware:1.8.5` (red).

i downgraded ocis to `1.8.3`, ...

**Comment by fschade** [comment]
same error for 1.8.3 :(

**Comment by fschade** [comment]
ok, expected error, wrong line

**Review by butonic — APPROVED** [review]

**Pull request merged**
